### PR TITLE
[fix] freediamter script has wrong version number.

### PR DESCRIPTION
**Pull request by jordanvrtanoski**
[lte][agw] Fixing a problem of the packet version for freediamter library 

## Summary

Packet version of the freediamter library is set to 0.0.1 while the AGW build script creates a dependancy to 1.2.0. This problem was discovered during porting to ARM.

## Test Plan

Generated the freediamter package. Package should show version 1.2.0
Install the generated package. Magma package should be able to install.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by tmdzk — APPROVED** [review]

**Comment by ssanadhya** [comment]
@rdefosse, @lionelgo could you please confirm if this change looks okay? Further, are you using the `oai-freediameter` package from `packages.magma.etagecom.io` in your pipeline?

**Comment by rdefosse** [comment]
@ssanadhya @lionelgo 

we are using the version defined in the dockerfile:

`https://github.com/magma/magma/blob/82fa960b13c73da983c8ae3fc0631fe18049412f/lte/gateway/docker/mme/Dockerfile.ubuntu18.04#L204`

The repo `https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter` has one commit ahead of Lionel's original repo

Raphael

**Comment by ekowtaylor** [comment]
@rdefosse this PR got approved, is it ready to be merged?

**Comment by ekowtaylor** [comment]
Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Comment by ekowtaylor** [comment]
@jordanvrtanoski any plans to move this forward?

**Comment by rdefosse** [comment]
@ekowtaylor 

no objections on my side.

Raphael

**Review by rdefosse — APPROVED** [review]

**Comment by ekowtaylor** [comment]
@tmdzk can you help get this merged

**Comment by themarwhal** [comment]
@jordanvrtanoski This PR is too old. (There are multiple required CI checks that cannot run) Can you rebase on the latest master and update the PR? Will remove the ready2merge label as it is not ready to be merged.

**Comment by ekowtaylor** [comment]
@jordanvrtanoski any updates?

**Comment by ekowtaylor** [comment]
Will need to rebase to move this forward

**Pull request closed without merging**
